### PR TITLE
patch: Update serv-vars.md

### PR DESCRIPTION
**Pull request by Jazzagi**
Update to change references of 'application' to 'fleet'

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

**Comment by Jazzagi** [comment]
Updated as part of #1987

File: pages/learn/manage/serv-vars.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Values can be up to 1MB (or approximately 1 million characters) in size each. No
 
 Environment and service variables defined at the fleet level are available to all devices in that fleet unless they are redefined with a variable of the same name at the device level.
```
Comment (chrisys) [inline pages/learn/manage/serv-vars.md:21]:
```suggestion
Variables defined at the fleet level are available to all devices in that fleet unless they are redefined with a variable of the same name at the device level.
```

File: pages/learn/manage/serv-vars.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ Values can be up to 1MB (or approximately 1 million characters) in size each. No
...
 
-They can be found from the application summary page under the *Environment Variables* and *Service Variables* tabs.
+They can be found from the fleet summary page under the *Environment Variables* and *Service Variables* tabs.
```
Comment (chrisys) [inline pages/learn/manage/serv-vars.md:23]:
```suggestion
They can be found from the fleet summary page under the *Variables* tab.
```

File: pages/learn/manage/serv-vars.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ You can override the value of a fleet variable by clicking *override* in the far
 The {{ $names.company.lower }} CLI and SDKs all include methods to easily read, add or update environment and service variables. Consult the appropriate reference for code examples.
 
 * [CLI environment reference](/tools/cli/#envs)
-* [Node.js SDK environment reference](/reference/sdk/node-sdk/#balena.models.application.envVar)
+* [Node.js SDK environment reference](/reference/sdk/node-sdk/#balena.models.fleet.envVar)
```
Comment (chrisys) [inline pages/learn/manage/serv-vars.md:64]:
```suggestion
* [Node.js SDK environment reference](/reference/sdk/node-sdk/#balena.models.application.envVar)
```

**Review by chrisys — APPROVED** [review]

**Comment by chrisys** [comment]
@balena-ci rebase

**Pull request merged**